api-reference/beta/api/administrativeunit_delete_scopedrolemembers.md

```diff
@@ -1,19 +1,19 @@
-# Delete scoped-role administrators
-Delete a scoped-role administrator from an adminstrative unit.
+# Remove a scopedRoleMember
+Remove a scoped-role member from an adminstrative unit.
 
 ## Prerequisites
-The following **scopes** are required to execute this API: *Directory.AccessAsUser.All*
+The following **scope** is required to execute this API: *Directory.AccessAsUser.All*
 
 ## HTTP request
 <!-- { "blockType": "ignored" } -->
 ```http
-DELETE /administrativeUnits/{id}/scopedAdministrators/{id}
+DELETE /administrativeunits/{id}/scopedrolemembers/{id}
```

Comment:
Should work, but this should be camel-cased:  scopedRoleMembers.  Please fix everywhere.